### PR TITLE
mkdir: add/refresh pages

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** latest

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM

File: pages/common/mkdir.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 > More information: <https://www.gnu.org/software/coreutils/mkdir>.
 
-- Create a directory in current directory or given path:
+- Create specific directories:
 
-`mkdir {{directory}}`
+`mkdir {{path/to/directory1 path/to/directory2 ...}}`
 
-- Create multiple directories in the current directory:
+- Create specific nested directories recursively:
```
Comment (navarroaxel) [inline pages/common/mkdir.md:10]: `nested` and `recursively` at the same time not sounds good for me. Based on the man page for `mkdir`:
```suggestion
- Create directories and their parents if needed:
```

File: pages/common/mkdir.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-> Creates a directory.
+> Create directories and set their permissions.
 > More information: <https://www.gnu.org/software/coreutils/mkdir>.
 
-- Create a directory in current directory or given path:
+- Create specific directories:
 
-`mkdir {{directory}}`
+`mkdir {{path/to/directory1 path/to/directory2 ...}}`
```
[the remark this answers is not in the document]
Comment (EmilySeville7cfg) [reply pages/common/mkdir.md:8]: There is no such syntax for directories now in our style guide. It should be proposed before this change can be applied.

[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/mkdir.md:8]: > If a command can take a variable number of arguments, use an ellipsis: {{arg1 arg2 ...}}. If one of the multiple options is possible, write it as {{either|or}}.

Yeah, This doesn't mention files and directories although we are suggesting them actively in PRs. I will open a PR soon to refresh style guide with some essential additions.

**Pull request merged**